frontend/src/metabase/nav/components/AppBar/AppBarSmall.styled.tsx

```diff
@@ -28,6 +28,10 @@ export const AppBarSubheader = styled.div<AppBarSubheaderProps>`
   transition: border-bottom-color 200ms ease;
   border-bottom: 1px solid
     ${props => (props.isNavBarOpen ? color("border") : "transparent")};
+
+  &:empty {
+    display: none;
+  }
```

Comment:
Without having to mess around with JS trying to hide the content, this seems more straight forward.

#### With this line
<img width="726" alt="Screenshot 2022-12-05 at 7 33 48 PM" src="https://user-images.githubusercontent.com/1937582/205727772-25ba8c10-c316-4db3-99d1-2a38f8ba2698.png">


#### Without this line
<img width="704" alt="Screenshot 2022-12-05 at 7 33 57 PM" src="https://user-images.githubusercontent.com/1937582/205727765-e59d889b-82ba-4453-b957-dc2c87ac401a.png">

Reply:
If I get it correctly, by fixing `isCollectionPathVisible` you won't need to do that here

Reply:
@ranquild that works. The thing was that there are so many booleans that affect if the breadcrumbs would be showing.

Seems the new approach is to just change how `isCollectionPathVisible` is calculated. And that's that.